AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginLazyLoadTests/LL1/GraphQLLazyLoadPostComment4V2Tests.swift

```diff
@@ -145,25 +145,21 @@ final class GraphQLLazyLoadPostComment4V2Tests: GraphQLLazyLoadBaseTest {
                   updatedAt
                   post {
                     id
+                    createdAt
+                    title
+                    updatedAt
                     __typename
                   }
                   __typename
                 }
               }
-              post {
-                id
-                createdAt
-                title
-                updatedAt
-                __typename
-              }
             }
             __typename
           }
         }
         """
         XCTAssertEqual(request.document, expectedDocument)
-        let createdComment = try await mutate(request)
+        _ = try await mutate(request)
```

Comment:
Do we need underscore, I think mutate is discardable result? 
```suggestion
try await mutate(request)
```